docs/docs/product-documentation.md

```diff
@@ -150,15 +141,133 @@ Figure 2: Product feature diagram (Swapchain, 2020c) [5]
+
+// Failure notification page 
+
+1. Something went wrong during the transaction processing. Funds previously sent to the HTLC are refunded. 
+
+
+**//Screencast - to be added** 
+
 
 ### 3.3. User guides
 
-***
+#### 3.3.1 How do I get started? 
+
+To trade cryptocurrency with Swapchain no user account is needed. You just enter the app and provide the needed information. 
+
+#### 3.3.2 What information do I need to trade? 
+
+- Your crpytocurrency address: Alphanumeric characters representing the destination for a payment.
+- Your public key: Part of the keys used for encryption and decryption of information. The public key enables message encryption only when the respective private key is used. Is of no value without the respective private key.
+- Your private key: Secret key that should not be revealed to anybody else then the key owner.
+
+#### 3.3.3 What parameters can I set? 
+
+- Bid Amount: The amount of currency you want to give away. 
+- Ask Amount: the amount of currency you want to receive. 
+- Bid Type: The type of cryptocurrency you want to give away (BTC/BTS). 
+- Ask Type: The type of cryptocurrency you want to receive (BTC/BTS). 
+- Transaction fee priority: Transaction fees you want to pay for your order to be processed. The lower the transaction fee the longer the transaction will take (Low/Medium/High priority and transaction fees respectively).
+- Timelock: The duration of the timelock being active on the HTLC. As soon as the timelock expires the currency from the pending transaction is refunded to the owner (Low/Medium/High duration of timelock). 
+
+#### 3.3.4 How do I initiate the swap? 
+
+1. Enter the Swapchain app and choose the proposer option in order to initiate a swap.
+2. Fill out the form with all the needed information and click Submit. 
+3. A secret hash is generated. Copy the hash and send it to your exchange partner together with your Bitcoin publickey and your Bitshares account name. 
+4. Stay in the browser until the transaction is completed. Do not leave the browser by any means. 
+5. After the the successful transaction or redemption because of timelock expiry you are notified and free to leave the browser. 
+
+#### 3.3.5 How do I complete the swap as a counterpart? 
+
+1. Enter the Swapchain app and choose the option to accept a swap.
+2. Enter the hash that was previously passed to you by your exchange partner that set up the HTLC.
+3. Sign the transaction with your provate key. 
+4. Do not leave the browser before receiving a notification of either a successful transaction or failure.
+
+#### 3.3.6 What happens if something goes wrong? 
+
+In case your exchange partner does not react to the swap you will be refunded your money as soon as the timelock expires. 
+In case of technical issues.... 
+//müssen wir tatsächlich noch mal mit den anderen besprechen, was dann passiert. Das weiß ich nicht so recht. was sind denkbare Fälle, die eintreten können und den Swap verhindern könnten? 
```

Comment:
In case of technical issues you may refund your Bitcoin HTLC manually. Your Bitshares HTLC gets refunded automatically.

Reply:
Kindly note that our software is MIT licensed and therefore comes without any warranty or liability.